Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -30,7 +30,7 @@ function Locate-TestVersion()
 	{
 		$regPath = "HKLM:\SOFTWARE\Wow6432Node\Microsoft\DevDiv\vstf\Servicing"
 	}
-	$keys = Get-Item $regPath | %{$_.GetSubKeyNames()}
+	$keys = Get-ChildItem $regPath | Where-Object {$_.GetSubKeyNames() -contains "testagentcore"}
```

Comment:
checked

---
In reply to: [68206532](https://github.com/Microsoft/vsts-tasks/pull/1934#discussion_r68206532) [](ancestors = 68206532)